rest/route/test.go

```diff
@@ -106,10 +106,13 @@ func (tgh *testGetHandler) Run(ctx context.Context) gimlet.Responder {
 			Limit:       tgh.limit,
 			Page:        page,
 		}
-		cedarTestResults, err := apimodels.GetCedarTestResults(ctx, opts)
+		cedarTestResults, status, err := apimodels.GetCedarTestResults(ctx, opts)
 		if err != nil {
 			return gimlet.MakeJSONInternalErrorResponder(errors.Wrap(err, "getting test results"))
 		}
+		if status != http.StatusOK && status != http.StatusNotFound {
```

Comment:
Should this propagate the HTTP status code from Cedar to the user rather than unconditionally return 500?

Reply:
Yeah I went back and forth about this, but it seems like its still an internal server error since it is failing to connect to a third-party service. I am not too opinionated on this though.